data/Dockerfiles/acme/docker-entrypoint.sh

```diff
@@ -18,18 +18,14 @@ while true; do
 		-f ${ACME_BASE}/acme/private/account.key \
 		-k ${ACME_BASE}/acme/private/privkey.pem \
 		-c ${ACME_BASE}/acme \
-		$*
+		${MAILCOW_HOSTNAME} autoconfig.${MAILCOW_HOSTNAME#*.} autoconfig.${MAILCOW_HOSTNAME#*.} ${ADDITIONAL_SAN}
 
 	case "$?" in
 		0) # new certs
 			# cp the new certificates and keys 
 			cp ${ACME_BASE}/acme/fullchain.pem ${ACME_BASE}/cert.pem
 			cp ${ACME_BASE}/acme/private/privkey.pem ${ACME_BASE}/key.pem
 
-			# also generate new dhparams with new certs
-			#(on the first run, the old dhparams may be the fakeoil from git)
-			openssl dhparam -out ${ACME_BASE}/dhparams.pem 2048
```

Comment:
@andryyy I've seen the explanation in the commit message about dhparams. Sorry for the noise.